wallet/src/db/tests.rs

```diff
@@ -13,6 +13,30 @@ impl HashMapDb {
     pub fn new(rc: Rc<RefCell<HashMap<Bytes, Bytes>>>) -> Self {
         Self { rc }
     }
+
+    #[allow(dead_code)]
```

Comment:
Why do we need to specify this flag to the linter?

Reply:
Because this function is not used anywhere, and I don't want to remove it because it feels weird to have an import but no export.